model/llm/llm.go

```diff
@@ -272,6 +312,66 @@ func (m *Model) RepairCode(ctx model.Context) (assessment metrics.Assessments, e
 	return assessment, nil
 }
 
+var _ model.CapabilityTranspile = (*Model)(nil)
+
+// Transpile queries the model to transpile source code to another language.
+func (m *Model) Transpile(ctx model.Context) (assessment metrics.Assessments, err error) {
+	transpileArguments, ok := ctx.Arguments.(*evaluatetask.TaskArgumentsTranspile)
+	if !ok {
+		return nil, pkgerrors.Errorf("unexpected type %#v", ctx.Arguments)
+	}
+
+	data, err := os.ReadFile(filepath.Join(ctx.RepositoryPath, ctx.FilePath))
+	if err != nil {
+		return nil, pkgerrors.WithStack(err)
+	}
+	stubFileContent := strings.TrimSpace(string(data))
+
+	data, err = os.ReadFile(filepath.Join(ctx.RepositoryPath, transpileArguments.OriginFilePath))
+	if err != nil {
+		return nil, pkgerrors.WithStack(err)
+	}
+	originFileContent := strings.TrimSpace(string(data))
+
+	importPath := ctx.Language.ImportPath(ctx.RepositoryPath, ctx.FilePath)
```

Comment:
Import path should come from the origin file :thinking: right?

Reply:
I think it should be  `FilePath` because it is the one that holds destination where we are going to write the generated code, right?

Reply:
Yes... Now that you changed where it is used in the prompt it's correct.